### PR TITLE
feat(perf-vitals): Add INP to allowed measurements

### PR DESCRIPTION
**Pull request by k-fish**
### Summary
This allows the `INP` measurement to be sent w/o counting towards the measurement limit.

### CODE REVIEW

**Review by jjbayer — APPROVED** [review]
I see this string already exists in the indexer table, otherwise we could have added it to the set of static strings as well. https://github.com/getsentry/sentry/blob/3d109cad5ca7e5083592a0419aa91fdbbcd06c08/src/sentry/sentry_metrics/indexer/strings.py#L48

**Comment by k-fish** [comment]
Tests need https://github.com/getsentry/relay/pull/1487 to be merged before they pass I think, since it's currently getting `none` instead of being cast to milliseconds.

File: src/sentry/sentry_metrics/indexer/strings.py [hunk 1]
```diff
@@ -71,6 +71,7 @@
     "d:transactions/breakdowns.span_ops.ops.browser@millisecond": PREFIX + 122,
     "d:transactions/breakdowns.span_ops.ops.resource@millisecond": PREFIX + 123,
     "d:transactions/breakdowns.span_ops.ops.ui@millisecond": PREFIX + 124,
+    "d:transactions/measurements.inp@millisecond": PREFIX + 125,
```
Comment (jjbayer) [inline src/sentry/sentry_metrics/indexer/strings.py:74]: Adding this string means that data collected before this change will be effectively lost for this metric, as any org-specific string index for this metric will now be obscured by the index defined here. Afaict this currently affects two orgs.

Comment (k-fish) [reply src/sentry/sentry_metrics/indexer/strings.py:74]: Thanks for taking a look again Joris 👍 

I guess I'm wondering a bit about the purpose these predefined metrics serve if moving forward we'll likely have to have mixed pre-defined (these static ones) and ones dynamically created already. Of course changing the test works too, but is this list now frozen in time? I imagine as more and more people use measurements, most measurement names we'll come with will already have been sent in, thereby making this list no-longer updatable.

Comment (k-fish) [reply src/sentry/sentry_metrics/indexer/strings.py:74]: I'll do your recommendation of changing the test code, for now though, that is just a question moving forward for this list in the metrics indexer.

Comment (jjbayer) [reply src/sentry/sentry_metrics/indexer/strings.py]: > is this list now frozen in time? I imagine as more and more people use measurements, most measurement names we'll come with will already have been sent in

Yes I think that is an accurate assessment, looking at

https://github.com/getsentry/sentry/blob/9f4ca288acbb722a39650f5c04efdaa81ae46a58/src/sentry/sentry_metrics/indexer/strings.py#L18-L19

If at some point we want to reduce storage cost by consolidating duplicate strings, I guess we'll need a migration strategy to migrate the data behind a specific string from per-org to shared. cc @onewland

**Pull request merged**